operations/research-and-development/product/technical-writing-team-handbook/writer-levels.md

```diff
@@ -0,0 +1,91 @@
+# Technial Writer Levels
+
+These levels serve to help Technical Writers assess their opportunities for career growth in the Techncial Writing function. The levels below follow a growth track in the art and craft of Technical Writing and not in a person management track.
+
+### Level 1: Technical Writer I
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br> <br>*Scope is the domain over which you are expected to show ownership.* | Executes to deliver high quality documentation and in-product text updates. |
```

Comment:
```suggestion
| Ownership | Scope: "What do I work on?" <br> <br>*Scope is the domain over which you are expected to demonstrate ownership.* | Executes to deliver high quality documentation and in-product text updates. |
```

Reply:
If you agree with this update, let's ensure all instances of this sentence are updated for consistency.

Reply:
This came from the template that HR provided so I am hesitant to change it here for the consistency sake of other departments.